style.css

```diff
@@ -0,0 +1,88 @@
+.clearfix::before,
+.clearfix::after {
+  content: "";
+  clear: both;
+  display: table;
+}
```

Comment:
You have a clearfix rule in place, but never use it in your HTML. I would recommend putting the Education, Expertise and Interests sections inside a `div` with this class name to make use of clearfix. 👍🏻